.github/workflows/emscripten.yml

```diff
@@ -61,5 +60,9 @@ jobs:
         run: |
           pyodide venv .venv-pyodide
           source .venv-pyodide/bin/activate
-          npm install pyodide@${{ env.PYODIDE_VERSION }}
-          node tools/emscripten/pytest_scikit_image_pyodide.js --pyargs skimage
+          pip install dist/*.whl
+          pushd doc
+          pip install pytest matplotlib
+          # Don't use the cache provider plugin, as it doesn't work with Pyodide
+          # right now: https://github.com/pypa/cibuildwheel/issues/1966
+          pytest -svra -p no:cacheprovider --pyargs skimage
```

Comment:
Regarding the pytest internal cache plugin failure, I'll go along with whatever you recommend.

Reply:
Thanks! I think we can go ahead with this solution, then. Resolving this conversation. :)